system-parachains/gluttons/glutton-kusama/src/lib.rs

```diff
@@ -88,7 +88,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("glutton"),
 	impl_name: create_runtime_str!("glutton"),
 	authoring_version: 1,
-	spec_version: 1_000_000,
+	spec_version: 1_001_000,
```

Comment:
I guess transaction version hasn’t changed for glutton?

Reply:
Correct